alpine.dockerfile

```diff
@@ -25,49 +25,24 @@ LABEL maintainer="Microsoft" \
       org.label-schema.vcs-url="https://github.com/Azure/azure-cli.git" \
       org.label-schema.docker.cmd="docker run -v \${HOME}/.azure:/root/.azure -it mcr.microsoft.com/azure-cli:$CLI_VERSION"
 
-# bash gcc make openssl-dev libffi-dev musl-dev - dependencies required for CLI
-# openssh - included for ssh-keygen
-# ca-certificates
 
-# curl - required for installing jp
-# jq - we include jq as a useful tool
```

Comment:
`jq` is a very useful tool and we frequently see users using `jq` to parse Azure CLI's output. Removing tools like this will cause a breaking change.